src/internal/connector/mockconnector/mock_data_message.go

```diff
@@ -5,13 +5,12 @@ import (
 	"fmt"
 	"testing"
 
-	absser "github.com/microsoft/kiota-abstractions-go/serialization"
-	js "github.com/microsoft/kiota-serialization-json-go"
+	"github.com/alcionai/clues"
+	serialization "github.com/microsoft/kiota-abstractions-go/serialization"
```

Comment:
Suggestion:

```suggestion
	kserialization "github.com/microsoft/kiota-abstractions-go/serialization"
```

Reply:
Oh, that shouldn't be aliased in the first place.